src/components/Navbar.tsx

```diff
@@ -8,30 +8,55 @@ import {
   MenuItem,
   ListItemIcon,
   Tooltip,
+  Popover,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
 } from '@mui/material';
 import GitHub from '@mui/icons-material/GitHub';
 import Article from '@mui/icons-material/Article';
 import Logout from '@mui/icons-material/Logout';
 import Login from '@mui/icons-material/Login';
 import Avatar from '@mui/material/Avatar';
 import { useAuth0 } from '@auth0/auth0-react';
+import NotificationsIcon from '@mui/icons-material/Notifications';
 import { enableAuth } from '../utils/constants';
 import packageJson from '../../package.json';
 import logo from '../assets/logo.png';
 
-function Navbar({ isLoggedIn, onLogin }: { isLoggedIn: boolean; onLogin: () => void }) {
+function Navbar({
+  isLoggedIn,
+  onLogin,
+  notifications,
+}: {
+  isLoggedIn: boolean;
+  onLogin: () => void;
+  notifications: string[];
+}) {
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const openAccountMenu = Boolean(anchorEl);
 
   const { user, logout } = useAuth0();
+  const [anchorMailEl, setAnchorMailEl] = useState<null | HTMLElement>(null);
```

Comment:
```suggestion
  const [anchorNotifEl, setAnchorNotifEl] = useState<null | HTMLElement>(null);
```

Since we switched out the mail icon for notification, it's good to update the relevant variables.